### DISCUSSION TITLE
Level for BT's in E5 Completion notice?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#310] [Q&A] [answered]
Hi,
We know that the specifications for notice subtypes E1-E5 are not complete, but can you please tell us if the BT's for Review in subtype E5 are on lot level or if it should be on general level but refer to different sections including lots?

When can we expect to have some more specifications and examples regarding E5?

![image](https://user-images.githubusercontent.com/79446518/214028865-32f9e61d-4443-4bbb-a392-0efa12b25aa8.png)

**Comment by YvesJo** [comment, marked as answer]
BT-786 is there to point to the sections of the publication and BG-714 is expected to be "on general level".
For timing about availability of further information about the extended forms, please see directly with the Helpdesk: ted@publications.europa.eu
